webapp/src/types/types.ts

```diff
@@ -58,16 +58,16 @@ export type CallsConfig = {
     ICEServers: string[],
     AllowEnableCalls: boolean,
     DefaultEnabled: boolean,
+    MaxCallParticipants: number,
```

Comment:
Going snake case makes the server code more complex as we need yet another config wrapper, not sure if it's worth it to be honest but let me know.

Reply:
Nah, it's fine. We're kind of stuck with it for now.